contract-tests/sse-contract-tests/src/main.cpp

```diff
@@ -38,6 +38,7 @@ int main(int argc, char* argv[]) {
         srv.add_capability("report");
         srv.add_capability("post");
         srv.add_capability("reconnection");
+        srv.add_capability("read-timeout");
```

Comment:
We were already passing the parameter along, just hadn't advertised the capability.